components/Hero/StoryHero.tsx

```diff
@@ -45,8 +59,49 @@ export const StoryHero = ({
     year: 'numeric',
   });
 
+  const cropSize = styles.imageCrops[aspectRatio];
+  const cropWidth = parseInt(cropSize?.split('x')[0], 10);
+  const cropHeight = parseInt(cropSize?.split('x')[1], 10);
+  const mobileCropSize = styles.mobileImageCrops[mobileAspectRatio];
+  const mobileCropWidth = parseInt(mobileCropSize?.split('x')[0], 10);
+  const mobileCropHeight = parseInt(mobileCropSize?.split('x')[1], 10);
+
+  const renderOneImage = !!filename && !mobileFilename && mobileAspectRatio === aspectRatio;
+  // Render 2 different images if there is both a hero image or a mobile image
+  // Or when there is no mobile image, but the mobile aspect ratio is different from the desktop aspect ratio
+  const renderTwoImages = (!!filename && !!mobileFilename) ||
+    (!!filename && !mobileFilename && mobileAspectRatio !== aspectRatio);
```

Comment:
Author can upload an optional mobile hero image and choose a different mobile aspect ratio. Depending on these conditions, we can choose to render one single image for both desktop/mobile, or to render 2 (one hidden on mobile, and one hidden on desktop)